Jetcaster/core/domain/src/main/java/com/example/jetcaster/core/model/EpisodeInfo.kt

```diff
@@ -43,3 +43,15 @@ fun Episode.asExternalModel(): EpisodeInfo =
         published = published,
         duration = duration,
     )
+
+fun EpisodeInfo.asDaoModel(): Episode =
+    Episode(
+        uri = uri,
+        title = title,
+        subtitle = subTitle,
+        summary = summary,
+        author = author,
+        published = published,
+        duration = duration,
+        podcastUri = ""
```

Comment:
Will this cause an issue? Should it be propagated from the Podcast info? Or maybe an episode needs a primary key link to a Podcast?

Reply:
Changed